setup.py

```diff
@@ -45,9 +45,10 @@
     extras_require={
         'dev': ['pytest', 'pytest-cov', 'sphinx', 'pandas', 'matplotlib'],
     },
-    # package_data={
-    #     'sample': ['package_data.dat'],
-    # },
+    include_pacakge_data=True,
```

Comment:
@TomAugspurger There is a typo here ("pacakge") that prevents Stitch from installing.